lib/MusicBrainz/Server/Controller/Role/Alias.pm

```diff
@@ -42,9 +44,26 @@ sub aliases : Chained('load') PathPart('aliases')
     my $m = $c->model($self->{model});
     my $aliases = $m->alias->find_by_entity_id($entity->id);
     $m->alias_type->load(@$aliases);
-    $c->stash(
```

Comment:
You can have as many `$c->stash` calls as you want, so can just leave this here and not have a separate `else` below.